projects/project-model.js

```diff
@@ -0,0 +1,12 @@
+
+
+module.exports = {
+	findName
+}
+
+function findName(db) {
```

Comment:
This is one way to query the database - pass in the db itself and use it for the query.

We normally just import the db file at the top - that way we know 100% that `db` is the database file itself

(If we called `findName("random string")` we'd get an error, because that string isn't a proper database file.)